website/content/docs/networking/cni.mdx

```diff
@@ -1,93 +1,72 @@
+  Learn how to install Container Network Interface (CNI) reference plugins, configure Nomad's bridge networking, and how to use CNI networks with Nomad jobs.
 ---
 
 # Container Network Interface (CNI)
 
-Nomad has built-in support for scheduling compute resources such as
-CPU, memory, and networking. Nomad's network plugin support extends
-this to allow scheduling tasks with purpose-created or specialty network
-configurations. Network plugins are third-party plugins that conform to the
-[Container Network Interface (CNI)][cni_spec] specification.
+This page show you how to install the [CNI reference plugins][cni-plugin-docs] on your Linux distribution and configure brdige networking on your Nomad clients. You can apply this guide's workflow to installing any plugin that complies with the [Container Network Interface (CNI) Specification][cni-spec], but you should verify plugin compatibility with Nomad before deploying in production.
 
-Network plugins need to be installed and configured on each client. The [Nomad
-installation instructions][nomad_install] recommend installing the [CNI
-reference plugins][cni_ref] because certain Nomad networking features, like
-`bridge` network mode and Consul service mesh, leverage them to provide an
-operating-system agnostic interface to configure workload networking.
+## Introduction
 
-Custom networking in Nomad is accomplished with a combination of CNI plugin
-binaries and CNI configuration files.
+The CNI specification defines a network configuration format for administrators. The configuration contains directives for both the orchestrator and the plugins to consume. Nomad implements custom networking through a combination of CNI reference plugin binaries and CNI configuration files. Networking features, like bridge network mode and Consul service mesh, leverage the CNI reference plugins to provide an operating-system agnostic interface to configure workload networking.
 
-## CNI reference plugins
+## Requirements
 
-The CNCF maintains a set of reference CNI plugins for basic network operations.
-These plugins are required for Nomad's `bridge` networking mode and for integrating
-with Consul service mesh.
+- You are familiar with [CNI reference plugins][cni-plugin-docs].
+- You are familiar with [how Nomad uses Container Network Interface (CNI) plugins for bridge networking](/nomad/docs/networking#bridge-networking).
+- You are running Nomad on Linux.
 
-See the Linux [post-install steps][cni_install] for installing CNI reference plugins.
+## CNI plugins and bridge networking workflow
 
-To use Nomad's [`transparent_proxy`][] feature, you will also need the
-[`consul-cni`][] plugin.
+Perform the following on each Nomad client:
 
-## CNI plugins
+1. [Install CNI reference plugins](#install-cni-reference-plugins).
+1. [Configure bridge module to route traffic through iptables](#configure-bridge-module-to-route-traffic-through-iptables).
+1. [Verify cgroup controllers](#verify-cgroup-controllers).
```

Comment:
cgroup controllers aren't relevant to CNI -- they're in the install docs but aren't related to networking.

```suggestion
```